core/src/test/java/apoc/example/ExamplesTest.java

```diff
@@ -23,6 +24,11 @@ public void setUp() throws Exception {
         TestUtil.registerProcedure(db,Examples.class);
     }
 
+    @After
+    public void tearDown() {
```

Comment:
should this be AfterClass instead as we don't create/start the db per test method. and doing it per method would explode the test runtime.